pkg/kubelet/rkt/rkt.go

```diff
@@ -52,11 +52,12 @@ import (
 )
 
 const (
-	RktType = "rkt"
+	RktType                      = "rkt"
+	DefaultRktAPIServiceEndpoint = "localhost:15441"
 
 	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "0.13.0"
-	recommendedRktBinVersion = "0.13.0"
+	minimumRktBinVersion     = "1.2.1"
```

Comment:
@woodbor So we probably won't support <1.2 anymore because those versions missed the other functions like injecting `--hostname` as well...
We shouldn't be so aggressive after we announce rktnetes 1.0, but for now supporting a very small number of versions makes our life much easier :)